### PR TITLE
Add rationale

### LINKED ISSUE #9: Add rationale
- what does the plugin do?
- how does it differ for using variables?

### PR DESCRIPTION
**Pull request by jeddy3**
[Rendered markdown](https://github.com/signal-noise/stylelint-scales/blob/e5bbc9653c8bd03b610ec9f49bbf0b915c06886f/README.md#why).

Closes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,3 +35,24 @@ Like so:
...
 - [`space`](./lib/rules/space/README.md): Specify a scale for space.
+
+Ref: [Styled System Keys Reference](https://styled-system.com/theme-specification#key-reference)
+
+# Why?
+
+This plugin can help you create:
+
+- a consistent look and feel for your apps
+- efficient collaboration you and the designers
```
Comment (marcelkornblum) [inline README.md]:
```suggestion
- efficient collaboration between you and the designers
```

File: README.md [hunk 2]
```diff
@@ -35,3 +35,24 @@ Like so:
...
+
+When designers review apps in the browser they generally use relative terms. For example, "The space between that heading and that paragraph feels too tight, can we make it bigger?"
+
+You can then pick the next value on the scale and be confident that it'll be consistent with the overall design.
```
Comment (marcelkornblum) [inline README.md]: probably also worth noting that the the development o the scale ought to include both disciplines, perhaps with a hat-tip to a constraints-based design authority?

Comment (jeddy3) [reply README.md]: Agreed. I'll put something in the list of rules section where we already talk about specifying scales.

**Comment by jeddy3** [comment]
I've updated the PR. I haven't mentioned "constraint-based design" by name as I'm struggling to find an authority on it.

**Pull request merged**